packages/data-union-solidity/contracts/DataUnionSidechain.sol

```diff
@@ -134,20 +134,18 @@ contract DataUnionSidechain is Ownable, IERC20Receiver, IERC677Receiver {
      * @param newDataUnionFee fee that goes to the DU beneficiary
      */
     function setFees(uint256 newAdminFee, uint256 newDataUnionFee) public onlyOwner {
-        require((newAdminFee + newDataUnionFee) <= 1 ether, "error_Fees");
+        require((newAdminFee + newDataUnionFee) <= 1 ether, "error_fees");
         adminFeeFraction = newAdminFee;
         dataUnionFeeFraction = newDataUnionFee;
         emit FeesSet(adminFeeFraction, dataUnionFeeFraction);
     }
 
-    function setDataUnionBeneficiary(address _dataUnionBeneficiary) public onlyOwner {
-        require(_dataUnionBeneficiary != address(0), "invalid_address");
```

Comment:
why was this removed? what happens if you use 0 as address?

Reply:
Then the beneficiary gets set to zero. There should be no special logic in place for zero beneficiaries (indirectly maybe if member 0x0000 were special, but it shouldn't be). This is onlyOwner function, and I see no reason zero were different from just random dummy address.